broadcastclient/broadcastclient.go

```diff
@@ -44,40 +45,48 @@ func FeedConfigAddOptions(prefix string, f *flag.FlagSet, feedInputEnable bool,
 
 var FeedConfigDefault = FeedConfig{
 	Output: wsbroadcastserver.DefaultBroadcasterConfig,
-	Input:  DefaultBroadcastClientConfig,
+	Input:  DefaultConfig,
 }
 
-type BroadcastClientConfig struct {
-	RequireChainId bool          `koanf:"require-chain-id"`
```

Comment:
Was this not used anywhere?

Reply:
Just changed to `broadcastclient.Config`